### ISSUE TITLE
qrcode-dependency throws an error on Android ("Execution failed for task ':qrcode:compileDebugKotlin'.")

### ISSUE THREAD

**Issue by mangowi** [deadblit/barras#7]
Hello,

Thank you for the nice package.

I've been playing around with it, and it was working fine. But after upgrading to new flutter. 

The package throws exception when building the project.  Execution failed for task ':qrcode:compileDebugKotlin'.

<img width="1386" alt="Screenshot 2021-01-01 at 14 06 27" src="https://user-images.githubusercontent.com/7607368/103439174-eb094700-4c3a-11eb-8fe8-c46cd962924b.png">
@deadblit 

Pub Dependencies 
<details>
`publish_to: 'none' 
  version: 1.0.0+1
  
  environment:
    sdk: ">=2.7.0 <3.0.0"
  
  dependencies:
    flutter:
      sdk: flutter
  
  
    cupertino_icons: ^1.0.1
    google_fonts: ^1.1.1
    firebase_messaging: ^7.0.3
    overlay_support: ^1.0.5
    url_launcher: ^5.7.10
    flutter_local_notifications: ^3.0.2
    webview_flutter: ^1.0.7
    http: ^0.12.2
    shared_preferences: ^0.5.12+4
    barras: ^0.0.2
  
  dev_dependencies:
    flutter_test:
      sdk: flutter
    integration_test:
      sdk: flutter

`
  
</details>

android/build.gradle [project root]

<details>
`
    buildscript {
        ext.kotlin_version = "1.3.50"
        repositories {
            google()
            jcenter()
        }
    
        dependencies {
            classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
            classpath 'com.android.tools.build:gradle:4.1.0'
            // Add the google services classpath
            classpath 'com.google.gms:google-services:4.3.2'
    
        }
    }
    
    allprojects {
        repositories {
            google()
            jcenter()
        }
    }
    
    rootProject.buildDir = '../build'
    subprojects {
        project.buildDir = "${rootProject.buildDir}/${project.name}"
    }
    subprojects {
        project.evaluationDependsOn(':app')
    }
    
    task clean(type: Delete) {
        delete rootProject.buildDir
    }

`
</details>

android/app/build.gradle 

<details>
`
    def localProperties = new Properties()
    def localPropertiesFile = rootProject.file('local.properties')
    if (localPropertiesFile.exists()) {
        localPropertiesFile.withReader('UTF-8') { reader ->
            localProperties.load(reader)
        }
    }
    
    def keystoreProperties = new Properties()
    def keystorePropertiesFile = rootProject.file('key.properties')
    if (keystorePropertiesFile.exists()) {
           keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
    }
    
    
    def flutterRoot = localProperties.getProperty('flutter.sdk')
    if (flutterRoot == null) {
        throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
    }
    
    def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
    if (flutterVersionCode == null) {
        flutterVersionCode = '1'
    }
    
    def flutterVersionName = localProperties.getProperty('flutter.versionName')
    if (flutterVersionName == null) {
        flutterVersionName = '1.0'
    }
    
    apply plugin: 'com.android.application'
    apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
    
    android {
        lintOptions {
            abortOnError false
            checkReleaseBuilds false //<- add this line
        }
        compileSdkVersion 30
    
        defaultConfig {
            multiDexEnabled true
            // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
            applicationId "com.mangowi.mpwa"
            minSdkVersion 23
            targetSdkVersion 30
            versionCode flutterVersionCode.toInteger()
            versionName flutterVersionName
        }
        signingConfigs {
           release {
              
           }
        }
        buildTypes {
            release {
                // TODO: Add your own signing config for the release build.
                // Signing with the debug keys for now, so `flutter run --release` works.
                signingConfig signingConfigs.release
            }
        }
        flavorDimensions "flavors"
        // product flavors
        productFlavors{
           
            demomain{
                dimension "flavors"
                applicationIdSuffix ".app.mainflavor"
                versionCode 6
                versionName "1.0.0"
            }
            demohumle{
                dimension "flavors"
                applicationIdSuffix ".app.secondflavor"
                versionCode 6
                versionName "1.0.0"
            }
    
    
        }
    
    }
    
    dependencies{
        implementation 'com.android.support:multidex:1.0.3'
    }
    
    flutter {
        source '../..'
    }
    
    apply plugin: 'com.google.gms.google-services'

`
</details>

**Comment by jinbe** [comment]
I'm having the same issue

**Comment by antonioreyna** [comment]
same here, any solution?

**Comment by ulmas97** [comment]
same here

**Comment by goodwind** [comment]
go into <project_root>/Android/build.gradle and change gradle version in dependencies section to 3.5.0

```
dependencies {
        classpath 'com.android.tools.build:gradle:3.5.0'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
```

**Comment by hussamDana92** [comment]
same here

**Comment by goodwind** [comment]
> same here

Just read my previous comment. It helps!

**Comment by deadblit** [comment]
Fixed in #18
